### ISSUE TITLE
[Issue]: Since upgrade, "Open Jellyfin" from tray app opens document folder

### ISSUE THREAD

**Issue by p37307** [jellyfin/jellyfin-server-windows#77]
### Please describe your bug

Since upgrading to jellyfin_10.8.8_windows-x64.exe from the repo, when I right-click the tray app and click 'Open Jellyfin', it opens the Documents folder on Windows 10 instead of Jellyfin in the browser.

### Jellyfin Version

10.8.0

### if other:

10.8.8

### Environment

```markdown
- OS: Edition	Windows 10 Pro, Version	22H2, OS build	19045.2251, Experience	Windows Feature Experience Pack 120.2212.4180.0
- Jellyfin.Windows.Tray version 1.2.0.0
```

### Please attach any browser or client logs here

-  Nothing in the logs and nothing in windows logs about it. 
- Let me know what other info you many need since this is regarding the tray app.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by anthonylavado** [comment]
Looking through a test install, I'm unable to replicate this issue. It's not the most ideal method to call explorer.exe with a URL in the code, but in all tests it's opened the default browser - Edge, Firefox, Chrome.

What's the current default browser on your system?

**Comment by p37307** [comment]
@anthonylavado  My current browser is Microsoft Edge, (Chromium) Version 109.0.1518.70 (Official build) (64-bit). I upgraded Jellyfin to version  10.8.9 (Windows X64) hoping the upgrade would solve the issue but the issue is still there. It is not an issue on my backup Windows Jellyfin server, which has the same version of Windows (right down to the version build number) as well as the same Edge build.


I set Google Chrome as default browser, restarted and it still occurs and I went back to Edge as my default browser.

Is there something in the Windows registry that could make this happen? I looked at Windows default app settings for Windows Explorer as well as Edge, including by protocol, to see if something was amiss and it appears there isn't.

**Comment by p37307** [comment]
After months, this issue went away. I only made two changes to my Windows 10 computer. I noticed my 'Documents Folder' in Windows had been moved from a custom location back to the Default profile location and I changed it back to my custom location and my Windows computer installed Cumulative Update for Windows 10 KB5022834 and .Net Framework update KB5022729. Maybe it reset some setting that was causing the tray app to open Documents folder. I don't know enough about the tray app to know it if writes anything to the registry or not and whether these two changes corrected it or not. It's fixed now. Marking closed.
